xbmc/Application.cpp

```diff
@@ -1873,12 +1873,9 @@ void CApplication::Render()
   if(!g_Windowing.BeginRender())
     return;
 
-  CDirtyRegionList dirtyRegions;
-
   // render gui layer
   if (!m_skipGuiRender)
   {
-    dirtyRegions = g_windowManager.GetDirty();
```

Comment:
did you check GetDirty()? is it const or does it change state? is it used elsewhere or can it be dropped?